Draft.md

```diff
@@ -62,15 +62,24 @@ keywords_en:
 
-Die Beurteilung einer Publikation ist eine traditionsreiche Form des wissenschaftlichen Diskurses. Entsprechend gibt es ein unausgesprochenes Verständnis davon, was eine Rezension umfassen sollte. Aus Redaktionsperspektive können zusätzlich konkrete Maßnahmen zur Qualitätssicherung ergriffen werden. Dazu gehören ausformulierte Hinweise zum Anfertigen von Rezensionen oder - wie es die Redaktion der Archäologischen Informationen praktiziert - Reviews von Rezensionen. Was sollte aber nun eine gute Rezension von Software für den Einsatz in der Archäologie ausmachen? Geht es allein um die digitale Operation, die die Software zur Lösung einer wissenschaftlichen Problemstellung beisteuert? Oder müssen Aspekte der Nutzbarkeit, der Nachhaltigkeit und Anschlussfähigkeit ebenfalls Beachtung finden? Welche Rolle spielen technische und rechtliche Aspekte, wie etwa die Dokumentation des Quellcodes oder Lizensierungsfragen, in der Besprechung? Und schließlich, stellt die Software vielleicht selbst einen wissenschaftlichen Beitrag dar? Was sind die Leistungen der Software-Entwickler, die eine Besprechung berücksichtigen sollte?
+In dieser Handreichung erläutern wir Kriterien, die eine gute Forschungssoftware ausmachen und für deren Bewertung von Bedeutung sein können. Außerdem empfehlen wir eine Vorgehensweise zur Softwarerezension. Dieser Beitrag ist als Impuls gedacht. Diskussionsbeiträge und Anregungen sind erwünscht. Dazu können die Autoren über die angegebenen Adressen kontaktiert werden oder Anmerkungen direkt auf GitHub unter [https://research-squirrel-engineers.github.io/DGUF_Leitfaden/](https://research-squirrel-engineers.github.io/DGUF_Leitfaden/) hinterlegt werden.
 
-In diesem Beitrag empfehlen wir eine Vorgehensweise zur Softwarerezension und sammeln Kriterien, die für die Bewertung einer Software von Bedeutung sein können. Wir konzentrieren uns dabei bewusst auf Aspekte, die spezifisch für die Besprechung von Software sind. Dies bringt mit sich, dass die einer Rezension innewohnende, zentrale Fragestellung nach dem wissenschaftlichen Wert des Besprechungsgegenstands nur gestreift wird. Denn: An diesem Punkt stimmen die Anforderung an eine Besprechung einer traditionellen Publikation und Software letztlich überein. Weiterhin werden aufgrund der Vielfalt und Vielartigkeit von Software nicht alle von uns hier gesammelten Kriterien in jeder Rezension von Bedeutung sein. Es bleibt dem Rezensent überlassen, Aspekte auszuwählen und Schwerpunkte zu legen.
+Die Beurteilung einer Publikation ist eine traditionsreiche Form des wissenschaftlichen Diskurses. Entsprechend gibt es ein unausgesprochenes Verständnis davon, was eine Rezension umfassen sollte. Aus Redaktionsperspektive können zusätzlich konkrete Maßnahmen zur Qualitätssicherung ergriffen werden. Dazu gehören Hinweise zum Anfertigen von Rezensionen oder - wie es die Redaktion der Archäologischen Informationen praktiziert - Reviews von Rezensionen.
```

Comment:
Redaktionsperspektive ist sehr verwirrend. Im ersten Moment klingt das als wären wir die Redaktion. Mit Grübeln könnte damit auch die Redaktion der DGUF gemeint sein. Das sollte unbedingt klarer formuliert werden.

Reply:
Das mit der Redaktionsperspektive kommt vom alten vorhergehenden Absatz. Ich habe nämlich nicht nur gekürzt, sondern auch viel umgestellt.
Wie wäre es denn stattdessen einfach mit >>Zusätzlich können weitere konkretere Maßnahmen zur Qualitätssicherung ergriffen werden. Dazu gehören Hinweise zum Anfertigen von Rezensionen oder ...<<

Reply:
Ich finde, dieser Satz zu Maßnahmen der Qualitätssicherung kann raus. Das ist ein Rest aus der Zeit, wo ich mich in das Thema "reingeschrieben" habe, leider schreibe ich immer erst viel, um dann zu streichen...